test/testcafe/spec/DeviceAssuranceGracePeriod_spec.js

```diff
@@ -95,13 +61,14 @@ test.requestHooks(dueByDaysMock)('should render correct messaging and navigate t
 
 test.requestHooks(oneOptionMock)('should render correct messaging for grace period with one option', async t => {
   const deviceAssuranceGracePeriodPage = await setup(t);
+  const expiryLocaleString = TimeUtil.formatDateToDeviceAssuranceGracePeriodExpiryLocaleString(new Date('2024-09-05T00:00:00.000Z'));
```

Comment:
Is there any way to avoid using the `TimeUtil` here in the test as well? I assume you are doing this so the string is always formatted for the TZ of the machine running the test rather than using a static string. Not ideal, but could be acceptable if we're unit testing `formatDateToDeviceAssuranceGracePeriodExpiryLocaleString` well

Reply:
There might be ways to override the timezone in testcafe, although forcing a timezone in a unit test is a lot easier.

Reply:
Yeah looks like TestCafe doesn't have a built-in way to enforce a specific timezone. During my first pass at this PR I was struggling to set a hard-coded timezone for Jest, but I just found a solution that works. Will add some unit tests